core/src/main/java/com/linecorp/armeria/internal/common/util/TemporaryThreadLocals.java

```diff
@@ -148,28 +161,25 @@ public StringBuilder stringBuilder() {
         }
     }
 
-    /**
-     * Switches the internal representation of the specified {@link StringBuilder} from LATIN1 to UTF16,
-     * so that character operations do not have performance penalty.
-     */
-    @SuppressWarnings("checkstyle:AvoidEscapedUnicodeCharacters")
-    private static StringBuilder inflate(StringBuilder stringBuilder) {
-        stringBuilder.append('\u0100');
-        stringBuilder.setLength(0);
-        return stringBuilder;
+    private void lock() {
+        assert !lock;
```

Comment:
But you can still add a message like before :wink: e.g. `assert !lock : "...";`

Reply:
Thanks 🤣 I thought no message is preferred from many `assert` cases in this project.

Reply:
Not really :grin: